ui/pages/home/home.component.js

```diff
@@ -330,6 +334,29 @@ export default class Home extends PureComponent {
             }
           />
         ) : null}
+
+        {removeCollectibleMessage === 'success' ? (
+          <ActionableMessage
+            type="success"
```

Comment:
We want this to be `type="danger"` so that it's red.

Reply:
Updated

<img width="787" alt="Screenshot 2023-01-20 at 11 25 05 AM" src="https://user-images.githubusercontent.com/39872794/213627238-b900ab74-1e1f-4eaa-9ed9-dde063234e62.png">